### PR TITLE
Remove code related to the Videopress TV and Videopress TV purchase flows

### PR DESCRIPTION
**Pull request by xavier-lc**
Related to https://github.com/Automattic/dotcom-forge/issues/9333

## Proposed Changes

* We are removing the "Videopress" flow (see [project thread](paYJgx-5ol-p2)), and while working on that I noticed the Videopress TV flow, which never got launched, so I'm removing it as well.

## Testing Instructions

* Open the live preview
* Go to `/setup/videopress-tv`
* Check that you are redirected to wp-admin
* Repeat the previous steps with `/setup/videopress-tv-purchase`

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-aUh-p2)?

### CODE REVIEW

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss [hunk 1]
```diff
@@ -159,8 +159,7 @@
 		}
 	}
 
-	&.videopress,
-	&.videopress-tv {
+	&.videopress {
```
Comment (rcrdortiz) [inline client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss:162]: Is this videopress selector still needed? Will we render videopress components?

File: client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss [hunk 2]
```diff
@@ -73,21 +73,7 @@
...
-		font-size: rem(38px); //typography-exception
-		font-weight: 700;
-		letter-spacing: -0.02em;
-	}
-}
-
-.is-videopress-stepper.is-processing-step,
-.is-videopress-tv-stepper,
-.is-videopress-tv-purchase-stepper {
+.is-videopress-stepper.is-processing-step {
```
Comment (rcrdortiz) [inline client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss:76]: Can we remove this?

File: client/landing/stepper/declarative-flow/internals/steps-repository/site-options/index.tsx
Comment (rcrdortiz) [inline client/landing/stepper/declarative-flow/internals/steps-repository/site-options/index.tsx:1]: Should we remove this component?

File: packages/onboarding/src/utils/flows.ts [hunk 3]
```diff
@@ -12,8 +12,6 @@ export const LINK_IN_BIO_POST_SETUP_FLOW = 'link-in-bio-post-setup';
 export const CONNECT_DOMAIN_FLOW = 'connect-domain';
 export const VIDEOPRESS_FLOW = 'videopress';
 export const VIDEOPRESS_ACCOUNT = 'videopress-account';
```
Comment (rcrdortiz) [inline packages/onboarding/src/utils/flows.ts:13]: Can't we remove these flows?

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss (hunk 1 above)
Comment (xavier-lc) [reply client/landing/stepper/declarative-flow/internals/steps-repository/intro/styles.scss:162]: I'm removing the `videopress` flow in a separate PR so there's less changes to review.

File: client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss (hunk 2 above)
Comment (xavier-lc) [reply client/landing/stepper/declarative-flow/internals/steps-repository/processing-step/style.scss:76]: I'm removing the `videopress` flow in a separate PR so there's less changes to review.

File: packages/onboarding/src/utils/flows.ts (hunk 3 above)
Comment (xavier-lc) [reply packages/onboarding/src/utils/flows.ts:13]: Yes, I'll remove them in a separate PR so there's less changes to review.

File: client/landing/stepper/declarative-flow/internals/steps-repository/site-options/index.tsx
Comment (xavier-lc) [reply client/landing/stepper/declarative-flow/internals/steps-repository/site-options/index.tsx:1]: I can remove it in the next PR.

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx [hunk 4]
```diff
@@ -79,7 +78,7 @@ const useIntroContent = ( flowName: string | null ): IntroContent => {
 		if ( isLinkInBioFlow( flowName ) ) {
 			return {
 				title: createInterpolateElement(
-					__( 'You’re 3 minutes away from<br />a stand-out Link in Bio site.<br />Ready? ' ),
+					__( 'You’re 3 minutes away from<br />a stand-out Link in Bio site.<br />Ready?' ),
```
Comment (dsas) [inline client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx:81]: This change introduces a new translatable string, which concerned me. Looking at our "similar string" matching though, it should be copied as-is, so it's not a concern.

**Comment by rcrdortiz** [comment]
I think we need a better redirect for this flow. In production, if we use this flow we're redirected to WordPress.com when logged out. What do you think about redirecting this flow to `/start` instead of wp-admin and `/devdocs/start`?

**Comment by xavier-lc** [comment]
> I think we need a better redirect for this flow. In production, if we use this flow we're redirected to WordPress.com when logged out. What do you think about redirecting this flow to `/start` instead of wp-admin and `/devdocs/start`?

Good point, addressed on https://github.com/Automattic/wp-calypso/pull/95751/commits/8bbd4d62440417ceed8541c4b9ca07fd9897f56c

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx (hunk 4 above)
Comment (xavier-lc) [reply client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx:81]: Right, the editor decided to adjust them on save. Plz tell me if they should be reverted to their original state.

**Review by rcrdortiz — APPROVED** [review]
I think we can move forward with this now.

Thanks for addressing the feedback!

File: client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx (hunk 4 above)
Comment (dsas) [reply client/landing/stepper/declarative-flow/internals/steps-repository/intro/index.tsx:81]: I think they're fine. My initial thoughts were that we'd have to wait for translations, but our translation system is clever enough to cope with this 👍

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/16950625).

**Pull request merged**